flask_resources/views/base.py

```diff
@@ -26,6 +27,7 @@ class BaseView(MethodView):
 
     resource_decorators = [
         request_loader,
+        url_args_parser,
```

Comment:
comment: I have the feeling that we are abusing decorators... now literally everything is a decorator... there is nothing wrong with calling functions on the methodview method...

Not just for this case, but in general on this python module we should scope when to use a decorator and when to use just a call to a method

Reply:
Ah, the reason we use decorators here so much, is that if we were using method calls, we would have to repeat the same method call in each of MethodView's http methods. And we *do* want to run the functionality in each of the http methods because the library is generic e.g. request arg parsing could occur on the list endpoints or the item endpoints.

Reply:
Shelved: Can still be encapsultated in a common function and we would avoid the "decorating per request" problem.